### ISSUE TITLE
HQM quest Bio Flight not detecting jetpack after it was used and refilled back in second part

### ISSUE THREAD

**Issue by alven4** [JasonMcRay/InfiTech-1.7#873]
#### Issue Description:
HQM quest Bio Flight won't detect jetpack filled with biogas after it was used and refilled back in the second part of the quest. I made sure that it wasn't in hover mode.

I added Twilight Forest and ChocoCraft.

#### Is the issue happening on server? Client? Both?
Server.

#### What happens:
Quest Bio Flight won't detect jetpack after it was used and refilled back in the second part of the quest.

#### What you expected to happen:
I expect the quest to detect jetpack in the second part of the quest.

#### Steps to reproduce (if aplicable):

1. Fill jetpack fully with biogas.
2. Use the jetpack, fill it back up and switch to normal mode.
3. Get to the second part of the quest Bio Flight.
4. Use Manual detect.

#### Modpack Version (Do *not* use word "latest", use the numerical indication of the version):
3.2.20

**Comment by lafflam** [comment]
I had the same problem.

I do not have any extra mods installed except for OptiFine.

Now that I read this post, I do remember removing the jetpack from the Fermenter prior to completely filling it because I ran out of biofuel. I decided to test it for a few seconds while making more fuel. Then I finished filling the jetpack.

Could wearing it once prevent the quest from registering so to that multiple players cannot get credit by just passing around one jetpack? I will make a new one and see if an unused jetpack registers in HQM properly.
